service/project_service/find.go

```diff
@@ -0,0 +1,22 @@
+package service
+
+import (
+	"unicode/utf8"
+
+	"github.com/go-errors/errors"
+	"github.com/torinos-io/api/type/model"
+)
+
+// FindRequest is a request object for finds project
+type FindRequest struct {
+	UUID string `json:"uuid"`
+}
+
+// Find returns project for given uuid
+func (s *service) Find(req *FindRequest) (*model.Project, error) {
+	if utf8.RuneCountInString(req.UUID) == 0 {
```

Comment:
Fix by introducing https://github.com/asaskevich/govalidator

(And you should use `len` over `utf8.RuneCountInString`.)